bootstore/src/trust_quorum/error.rs

```diff
@@ -4,17 +4,20 @@
 
 //! Error type for trust quorum code
 
+use derive_more::From;
 use thiserror::Error;
 
-#[derive(Debug, Error)]
+#[derive(From, Debug, Error)]
 pub enum TrustQuorumError {
     // Temporary until the using code is written
-    #[allow(dead_code)]
-    #[error("Not enough peers to unlock storage")]
-    NotEnoughPeers,
-
-    // Temporary until the using code is written
-    #[allow(dead_code)]
     #[error("Rack secret construction failed: {0:?}")]
-    RackSecretConstructionFailed(vsss_rs::Error),
+    Vsssrs(vsss_rs::Error),
+
+    // Failed to encrypt something
+    #[error("Failed to encrypt")]
+    FailedToEncrypt,
+
+    // Failed to decrypt something
+    #[error("Failed to encrypt")]
```

Comment:
```suggestion
    #[error("Failed to decrypt")]
```

Reply:
Fixed in [671d4b1](https://github.com/oxidecomputer/omicron/pull/3167/commits/671d4b1c7b8cf395db8f25beeadc013668f9e594)